### PR TITLE
Split mimpid into major, minor, patch parts

### PR DESCRIPTION
**Pull request by Silabs-ArjanB**
Signed-off-by: Arjan Bink <Arjan.Bink@silabs.com>

### CODE REVIEW

File: docs/user_manual/source/control_status_registers.rst [hunk 1]
```diff
@@ -1611,8 +1611,22 @@ Detailed:
 +-------------+-----------+------------------------------------------------------------------------+
 |   Bit #     |  R/W      |   Description                                                          |
 +=============+===========+========================================================================+
-| 31:0        | R         | Machine Implementation ID **mimpid_i**, see  :ref:`core-integration`   |
+| 31:20       | R (0x)    | Hardwired to 0.                                                        |
```
Comment (silabs-oysteink) [inline docs/user_manual/source/control_status_registers.rst:1614]: R (0x0) ?

File: docs/user_manual/source/control_status_registers.rst [hunk 2]
```diff
@@ -1611,8 +1611,22 @@ Detailed:
...
 +-------------+-----------+------------------------------------------------------------------------+
+| 19:16       | R (0x)    | **MAJOR**.                                                             |
```
Comment (silabs-oysteink) [inline docs/user_manual/source/control_status_registers.rst:1616]: Same as above, should this be "R (0x0)" ?

**Review by silabs-oysteink — CHANGES_REQUESTED** [review]

File: docs/user_manual/source/control_status_registers.rst (hunk 2 above)
Comment (Silabs-ArjanB) [reply docs/user_manual/source/control_status_registers.rst]: Should be fixed now

File: docs/user_manual/source/control_status_registers.rst (hunk 1 above)
Comment (Silabs-ArjanB) [reply docs/user_manual/source/control_status_registers.rst]: Should be fixed now

**Review by silabs-oysteink — APPROVED** [review]

**Pull request merged**